common/test/common/BoxTest.scala

```diff
@@ -0,0 +1,16 @@
+package common
```

Comment:
This is copypasta from https://github.com/guardian/box/blob/master/src/test/scala/com/gu/BoxSpec.scala